spec/02-integration/05-proxy/14-server_tokens_spec.lua

```diff
@@ -430,6 +470,8 @@ describe("headers [#" .. strategy .. "]", function()
           assert.res_status(200, res)
           admin_client:close()
 
+          ngx.sleep(1)
```

Comment:
Is this needed, or is it a leftover from dev? If needed, do we have an explanation about why?

Reply:
I had `lua-resty-worker-events 2.0.0` on my machine where it was required, but it is not required without it in current master, so I removed it.